src/components/overview/overview.tsx

```diff
@@ -0,0 +1,38 @@
+import * as React from "react";
+import { Link } from "react-router-dom";
+import { Button, Form, Input } from "semantic-ui-react";
+
+import { ApiService } from "../../services/apiService";
+import Filelist from "./feedback/Filelist";
+const apiService: ApiService = new ApiService();
+
+export default class Overview extends React.Component<any, any> {
+    public state = {
+        files: apiService.getAllDocumentsWithFeedback(),
+        selectedHash: "",
+    };
+    public selectFeedback = (hash) => {
+        this.setState({
+            selectedHash: hash,
+        });
+    };
+
+    public render() {
+        return (
+            <div className="">
+                <h1 className="">Feedback - Blockr</h1>
+                <div className="Filelist">
```

Comment:
Change this to comply with naming conventions.

```suggestion
                <div className="fileList">
```